solution/backend/serverless.yml

```diff
@@ -233,7 +233,8 @@ resources:
         Engine: aurora-postgresql
         EngineVersion: "15.2"
         DatabaseName: 'eregs'
-        BackupRetentionPeriod: 3
+        BackupRetentionPeriod: 7
+        BackupRetentionMinimum: 7
```

Comment:
I dont see this paramater in the documenation anywhere BackupRetentionMinimum.

Reply:
Go to https://docs.aws.amazon.com/securityhub/latest/userguide/rds-controls.html#rds-11 it lists the parameter here. Then click on "AWS Config rule: [db-instance-backup-enabled](https://docs.aws.amazon.com/config/latest/developerguide/db-instance-backup-enabled.html)" and it'll show the list of possible params.

Reply:
Ok, this seems with an out of sync thing with AWS documentation so it makes me a little bit on the "make sure this works correctly".  When pushed be on the look out if it recognizes the paramater.